src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5254,12 +5279,13 @@ namespace OutputReportTabular {
             ZonePreDefRep(iZone).SHGSAnLiteAdd += ZnRpt(iZone).LtsTotGain * mult * timeStepRatio;
             // HVAC Input Sensible Air Heating
             // HVAC Input Sensible Air Cooling
-            if ((ZnAirRpt(iZone).SumMCpDTsystem + ZnAirRpt(iZone).SumNonAirSystem) > 0.0) {
-                ZonePreDefRep(iZone).SHGSAnHvacHt +=
-                    ZnAirRpt(iZone).SumMCpDTsystem * TimeStepSys * SecInHour + ZnAirRpt(iZone).SumNonAirSystem * mult * TimeStepSys * SecInHour;
+            ZoneEqHeatorCool(iZone) =
+                ZnAirRpt(iZone).SumMCpDTsystem * TimeStepSys * SecInHour + ZnAirRpt(iZone).SumNonAirSystem * mult * TimeStepSys * SecInHour 
+                - ATUDeltaHeat(iZone) - ATUDeltaCool(iZone);
```

Comment:
If `TimeStepSys * SecInHour` is removed from this line, then `ATUHeat` and `ATUCool` [W] can be used here and we don't need the new `ATUDeltaHeat` and `ATUDeltaCool` variables.
Also, `ZoneEqHeatorCool` is just used locally in this loop, so it does not need to be an array.

Reply:
@mjwitte Just a cautionary check: should the `ZnAirRpt(iZone).SumMCpDTsystem` be multiplied by a zone/zone-list multiplier `mult` the same way as `ZnAirRpt(iZone).SumNonAirSystem` does in Line 5283 (and in a few other places across that file as well)? Or the multiplier for the `SumMCpDTsystem` has been taken care of someplace else?

Reply:
That's a very good question, this can be quite confusing. Everything related to air-side HVAC should already be multiplied, `SumMCpDTsystem` as calculated in  [`ZoneTempPredictorCorrector::CalcZoneComponentLoadSums`](https://github.com/NREL/EnergyPlus/blob/7badc378a280879046ae72d47b0b12e90e5ea128/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L6497) is the full value (already multiplied).  But `CalcZoneComponentLoadSums` does not account for multipliers when [calculating the optional zone heat balance imbalance](https://github.com/NREL/EnergyPlus/blob/7badc378a280879046ae72d47b0b12e90e5ea128/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L6672-L6673). Hmmm. And the I/O Reference for Zone Air Heat Balance System Air Transfer Rate says "This field is not multiplied ... `. 

Looks like `SumMCpDTsystem` should be divided by the zone multipliers the same way it is in [`CalcZoneSums`](https://github.com/NREL/EnergyPlus/blob/7badc378a280879046ae72d47b0b12e90e5ea128/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L6252-L6253). I wouldn't touch that for this PR. I would make some test files with and without multipliers (fully autosized) to prove that it's wrong for the Zone Air Heat Balance System Air Transfer Rate output variable and wrong for the imbalance ( Output:Diagnostics, DisplayZoneAirHeatBalanceOffBalance;) then post a new issue to clean this up. If I'm following everything correctly, the zone sensible heat gains report should work fine with or without multipliers as it is now.

Reply:
@mjwitte Thanks! Yes, this new issue will not affect the fix in the current PR. The code revisions for the original comments are done now.